pkg/apis/validation/validation.go

```diff
@@ -113,3 +115,11 @@ func keyEncodingValue(path *field.Path, s string) field.ErrorList {
 	}
 	return nil
 }
+
+func keystoreTypeValue(path *field.Path, s string) field.ErrorList {
+	if s != "PKCS12" {
+		return field.ErrorList{field.NotSupported(path, s, []string{"PKCS12"})}
```

Comment:
👍 slight bit but not essential, perhaps define a slice of 'valid values' here so we can reuse it in the error message instead of hard coding it twice?